patches.json

```diff
@@ -229,6 +229,9 @@
     },
     "FPC is getting disabled during deployments": {
       ">=2.3.2 <2.4.0": "MAGECLOUD-5069__fpc_is_getting_disabled_during_deployments__2.3.2.patch"
```

Comment:
patch for FPC was already updated in develop branch. Please merge those changes

Reply:
Resolved